src/dsl.ts

```diff
@@ -27,7 +27,7 @@ import { genericEntityRuleImpl } from "@useoptic/api-checks/build/sdk/generic-en
 import { ShouldOrMust } from "@useoptic/api-checks/build/sdk/types";
 import { jsonPointerHelpers } from "@useoptic/json-pointer-helpers";
 
-type SnykStablity = "wip" | "experimental" | "beta" | "ga";
+type SnykStablity = "wip" | "experimental" | "beta" | "ga" | "deprecated";
```

Comment:
`deprecated` is a lifecycle stage; not something an API can declare in its OpenAPI spec with an extension.